### ISSUE TITLE
Charterer 1 Android Build error MSB3231 Unable to remove directory

### ISSUE THREAD

**Issue by km6wsy** [xamarin/xamarin-forms-book-samples#54]
Charpter1 Andriod build with error

MSB3231: Unable to remove directory "obj\Debug\81\android\bin\classes". Could not find a part of the path 'AccessibilityManagerCompat_AccessibilityStateChangeListenerImplementor.class'.		 		

Have tried to clean and build, clean and rebuild solution and PlatformVisuals.Droid project, manual delete "obj\Debug\81\android\bin\classes" files and restored NuGet package, the problem still persists.

**Comment by meckblack** [comment]
Please I am having the same error, kindly comment if you have found a solution

**Comment by sunil-c** [comment]
You have to delete the Bin and Obj folders in the android project then build again

**Comment by apacha** [comment]
In my case the problem was that I hit the 260 character limit of Windows. So manually deleting the obj folder did not fix the problem. But moving the project into a directory with a shorter path did.

**Comment by GBParzival** [comment]
> In my case the problem was that I hit the 260 character limit of Windows. So manually deleting the obj folder did not fix the problem. But moving the project into a directory with a shorter path did.

-It worked!Thanks
